numpy/lib/tests/test_type_check.py

```diff
@@ -360,6 +360,15 @@ def test_generic(self):
         assert_(vals[1] == 0)
         assert_all(vals[2] > 1e10) and assert_all(np.isfinite(vals[2]))
         assert_equal(type(vals), np.ndarray)
+        
+        # perform the same tests but with nan, posinf and neginf keywords
+        with np.errstate(divide='ignore', invalid='ignore'):
+            vals = nan_to_num(np.array((-1., 0, 1))/0., 
+                              nan=10, posinf=20, neginf=30)
+        assert_all(vals[0] == 30) and assert_all(np.isfinite(vals[0]))
+        assert_(vals[1] == 10)
+        assert_all(vals[2] == 20) and assert_all(np.isfinite(vals[2]))
```

Comment:
Can these three lines be replaced by `assert_equal(vals, [30, 10, 20])` ?

Reply:
Oh I see, you were just following the pattern above. Still, I think it's better to use the shorter version if there is no problem.

Reply:
Yes, I was following the previous code.

No problem to update this.